roles/ogm/tasks/script_and_cron.yml

```diff
@@ -0,0 +1,21 @@
+---
+- name: Deploy backup script
+  ansible.builtin.template:
+    src: templates/ogm_backup.sh.j2
+    dest: ~umcg-ogm-dm/ogm_backup.sh
+    owner: umcg-ogm-dm
+    group: umcg-ogm-dm
+    mode: '0750'
+  become: true
+
+- name: 'Create cron job to make daily backup with the script'
+  ansible.builtin.cron:
+    name: Daily backup of OGM - deployed with ansible do not change anything!
```

Comment:
```suggestion
    name: Backup_OGM_data  # Unique ID required to update existing cronjob: do not modify.
```

Reply:
I suggest to keep those cronjob names short and "generic". Hence remove "daily" from the name, because otherwise if the interval changes either you keep "daily" in the name, which will be confusing, or you change the name and then Ansible cannot update the existing cron job.

Reply:
Agreed